data/templates/css/wikihow.scss

```diff
@@ -266,31 +261,17 @@ h2 + ol, h3 + ol, .steps_list_2 {
     }
 }
 
-.disclaimer {
-    width: 100%;
-    font: 400 0.8rem $font-secondary;
-    text-align: center;
-    padding-top: 40px;
-    padding-bottom: 40px;
-    margin-top: 40px;
-    border-top: 1px solid $gray-lighter;
-    clear: both;
-    p {
-        width: 100% !important;
-    }
-}
-
```

Comment:
I noticed you added this earlier, maybe it would better to aggregate these changes?